modules/fedimint-wallet-tests/tests/tests.rs

```diff
@@ -508,7 +512,9 @@ async fn peg_ins_that_are_unconfirmed_are_rejected() -> anyhow::Result<()> {
         .await;
     let block_count = dyn_bitcoin_rpc.get_block_count().await?;
     sync_wallet_to_block(
-        &mut dbtx.dbtx_ref_with_prefix_module_id(module_instance_id),
+        &mut dbtx
+            .to_ref_with_prefix_module_id(module_instance_id)
+            .into_non_committable(),
```

Comment:
Comparing to lifetime version, this is a bit worse. Lifetimes figured it out automatically, while here we need to convert the type somehow. We could `DerefMut` from commitable to non-commitable maybe.